ansible/server-state-playbooks/www/requirements.yml

```diff
@@ -16,9 +13,6 @@
   src: https://github.com/openmicroscopy/ansible-role-jekyll-build.git
   version: 1.2.0
 
-- name: openmicroscopy.lvm-partition
```

Comment:
It won't be used by a sub-role I'd have thought, given it makes system-wide changes. Certainly not here.